application/client/src/app/ui/views/sidebar/search/providers/providers.ts

```diff
@@ -485,6 +488,9 @@ export class Providers {
                     });
                 },
             });
+
+        this.contextMenuOptions(event.items);
+
```

Comment:
It would be better to put at the end, to show menu items (related to export/import) and the end of context-menu.